docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,539 @@
+    ret = noti_ex_item_group_add_child(group_item, image_item1);
+    ret = noti_ex_item_group_add_child(group_item, image_item2);
+    ret = noti_ex_item_group_add_child(group_item, button_item);
+    ret = noti_ex_item_group_add_child(group_item, selector);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+**Figure: Expanded Notification**
+
+![Expanded Notification](./media/expanded_notification.png)
+
+<a name="messaging"></a>
+## Creating Messaging Style Notification
+
+Since messages with different values can be added to a single notification, you can either show various messages in a single notification or you can configure it like a chat-room.
+You can also configure notifications using text items, image items, and time items:
+
+    ```
+    noti_ex_item_h group_item = NULL;
+    noti_ex_item_h chatmessage_item = NULL;
+    noti_ex_item_h name_handle = NULL;
+    noti_ex_item_h text_handle = NULL;
+    noti_ex_item_h image_handle = NULL;
+    noti_ex_item_h time_handle = NULL;
+    time_t current_time;
+
+    ret = noti_ex_item_group_create(&group_item, "group_id");
+
+    noti_ex_item_text_create(&name_handle, "name_id", "name", NULL);
+    noti_ex_item_text_create(&text_handle, "text_id", "text", NULL);
+    noti_ex_item_image_create(&image_handle, "image_id", APP_IMAGE_FULL_PATH);
+
+    time(&current_time);
+    noti_ex_item_time_create(&time_handle, "time_id", current_time);
+
+    ret = noti_ex_item_chat_message_create(&chatmessage_item, "message_id", name_handle,
+        text_handle, image_handle, time_handle, NOTI_EX_ITEM_CHAT_MESSAGE_TYPE_USER);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+
+    ret = noti_ex_item_group_add_child(group_item, chatmessage_item);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+**Figure: Messaging Style Notification**
+
+![Messaging Style Notification](./media/messaging_notification.png)
+
+<a name="reporter"></a>
+## Creating Notification-ex Reporter Handle
+
+To send a notification, you need to create a handle for the reporter. While creating a handle, you can register an error callback and an event callback:
+    ```
+    noti_ex_reporter_h handle;
+    noti_ex_reporter_events_s ev;
+
+    ev.event = _reporter_events_event_cb;
+    ev.error = _reporter_events_error_cb;
+
+    ret = noti_ex_reporter_create(&handle, ev, NULL);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+<a name="post"></a>
+## Posting Notification
+
+You can post notifications. You can also post notifications as a list.
+
+To post a notification, use `noti_ex_reporter_post()`:
+
+    ```
+    ret = noti_ex_reporter_post(handle, group_item, &request_id);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+To post the notification list, use `noti_ex_reporter_post_all()`:
+
+    ```
+    ret = noti_ex_reporter_post_list(handle, noti_list, 2, &request_id);
```

Comment:
`noti_ex_reporter_post_all()` in the description.
`noti_ex_reporter_post_list()` in the sample code.
Would you please check?